packages/theme/src/store/dashboard/posts.ts

```diff
@@ -55,7 +55,7 @@ export const useDashboardPosts = defineStore("dashboardPosts", () => {
     );
     if (postIdx === -1) return;
 
-    Object.assign(posts.value[postIdx], post);
+    if (posts.value[postIdx]) Object.assign(posts.value[postIdx], post);
```

Comment:
Same as [this](https://github.com/logchimp/logchimp/pull/1447/files#r2506840990)